.github/workflows/msal-angular-e2e.yml

```diff
@@ -33,10 +33,6 @@ jobs:
       fail-fast: false
       matrix:
         sample:
-          - 'angular9-v2-sample-app'
```

Comment:
@jo-arroyo do we need to create new/specific tests for the more recent angular releases?

Reply:
@hectormmg are there files we want to remove as well?  Or is that in a later PR?

Reply:
I think tests for the more recent angular releases can be done when we do msal-angular v3

Reply:
@hectormmg we could remove samples for unsupported versions and keep them in the v2 branch once we ship v3. Having said that, when we eventually deprecate v2 the samples will be permanently deleted. Does that sound okay?